protobuf/v1/nexus.proto

```diff
@@ -50,6 +50,8 @@ service NexusRpc {
   rpc GetRebuildState (RebuildStateRequest) returns (RebuildStateResponse) {}
   rpc GetRebuildStats (RebuildStatsRequest) returns (RebuildStatsResponse) {}
   rpc GetRebuildHistory (RebuildHistoryRequest) returns (RebuildHistoryResponse) {}
+  rpc ListRebuildHistory (google.protobuf.Empty) returns (RebuildHistoryListResponse) {}
```

Comment:
let's add a request message with limit of rebuilds per nexus

Reply:
I thought that can be done in the API implementation. We cant limit records in protobuf right?

Reply:
Incorporated

Reply:
This is optional, If not passed or passed 0 we set it to return all. That can be changed in API implementation to have a default number as well.